Firestore/core/include/firebase/firestore/geo_point.h

```diff
@@ -55,6 +58,16 @@ class GeoPoint {
     return longitude_;
   }
 
+  /**
+   * Returns a string representation of this `GeoPoint` for logging/debugging
+   * purposes.
+   *
+   * Note: the exact string representation is unspecified and subject to change;
+   * don't rely on the format of the string.
+   */
+  std::string ToString() const;
+  friend std::ostream& operator<<(std::ostream& out, const GeoPoint& geo_point);
```

Comment:
Nit: the implementation doesn't seem to be accessing any private data members, does it have to be a `friend`?

Reply:
This rationale makes sense to me. Feel free to keep as it was, and sorry about the trouble.